be/src/Project/Services/Client/Client.Api/Services/Clients/ClientsService.cs

```diff
@@ -29,45 +30,77 @@ public class ClientsService : IClientsService
 
         public async Task<PagedResults<IEnumerable<ClientServiceModel>>> GetAsync(GetClientsServiceModel model)
         {
-            var clients = from c in this.context.Clients
-                          where c.SellerId == model.OrganisationId.Value && c.IsActive
-                          select new ClientServiceModel
-                          {
-                              Id = c.Id,
-                              Name = c.Name,
-                              Email = c.Email,
-                              CommunicationLanguage = c.Language,
-                              PhoneNumber = c.PhoneNumber,
-                              LastModifiedDate = c.LastModifiedDate,
-                              CreatedDate = c.CreatedDate
-                          };
+            var clients = this.context.Clients.Where(x => x.IsActive);
 
-            if (!string.IsNullOrWhiteSpace(model.SearchTerm))
+            if (string.IsNullOrWhiteSpace(model.SearchTerm) is false)
             {
                 clients = clients.Where(x => x.Name.StartsWith(model.SearchTerm));
             }
 
             clients = clients.ApplySort(model.OrderBy);
 
-            return clients.PagedIndex(new Pagination(clients.Count(), model.ItemsPerPage), model.PageIndex);
+            var pagedResults = clients.PagedIndex(new Pagination(clients.Count(), model.ItemsPerPage), model.PageIndex);
+
+            var pagedClientServiceModel = new PagedResults<IEnumerable<ClientServiceModel>>(pagedResults.Total, pagedResults.PageSize);
+
+            var clientsList = new List<ClientServiceModel>();
+
+            foreach (var client in pagedResults.Data.OrEmptyIfNull().ToList())
+            {
+                var item = new ClientServiceModel
+                {
+                    Id = client.Id,
+                    Name = client.Name,
+                    Email = client.Email,
+                    CommunicationLanguage = client.Language,
+                    PhoneNumber = client.PhoneNumber,
+                    LastModifiedDate = client.LastModifiedDate,
+                    CreatedDate = client.CreatedDate
+                };
+
+                var clientGroups = this.context.ClientsGroups.Where(x => x.ClientId == client.Id && x.IsActive);
```

Comment:
**suggestion:** it can be changed to item.Groups = this.context.ClientsGroups.Where(x => x.ClientId == client.Id && x.IsActive).Select(x => x.GroupId)